developer_manual/exapp_development/tech_details/api/textprocessing.rst

```diff
@@ -0,0 +1,72 @@
+===============
+Text-Processing
```

Comment:
@marcelklehr can we not include Text-Processing API into documentation, as it is deprecated?

Reply:
Yeah, you can leave it out

Reply:
@marcelklehr same for Speech-To-Text and Machine Translation APIs as well?

Reply:
Yeah, they're all deprecated in server anyway.